docs/integration/vercel-logs-integration.md

```diff
@@ -0,0 +1,63 @@
+title: Vercel Logs Integration
+description: With our new Vercel Logs Integration you get insight into your whole Vercel account. View logs from all your projects, serverless functions, static and Jamstack websites, including build logs and errors!
+
+By configuring a Sematext Log Drain Integration to point to Sematext Logs you can get insight into your whole Vercel account in one place!
+
+## Vercel Logs Quick Start
+
+The Sematext Vercel Logs Integration collects logs from the following:
+
+- All Builds
+- Static Websites
+- Jamstack Websites
+- Serverless Functions that succeed
+- Serverless Functions that fail, crash, or cause an error
+- Serverless Functions that are cold starts
+
+We give you 7 reports out-of-the-box. You can add as many additional custom reports and charts as you want!
+
+## Install the Sematext Logs Vercel Integration
+
+Install the [Sematext Logs integration from the Vercel Integrations Marketplace](https://vercel.com/integrations/sematext-log-drain) to add it to your Vercel account.
+
+![](https://vercel.com/docs/static/guides/debugging-and-troubleshooting-vercel-logs-with-sematext/step-2-sematext-log-drain.png)
+
+Select the account or team your deployed project belongs to. You will then be redirected to Sematext Logs.
+
+![](https://vercel.com/docs/static/guides/debugging-and-troubleshooting-vercel-logs-with-sematext/step-2-add-sematext-log-drain.png)
+
+## Create a Vercel Logs App
+
+In the Sematext UI, [create a Vercel App](https://apps.sematext.com/ui/logs-create) ([click here for our EU region](https://apps.sematext.com/ui/logs-create)). This will set up a Logs App that acts as container for your data with prebuilt reports and charts for easy troubleshooting.
```

Comment:
click here for our EU region](https://apps.  => click here for our EU region](https://apps.eu

Reply:
Ah damn, fixing.